components/post_view/post_body/post_body.tsx

```diff
@@ -151,7 +156,8 @@ export default class PostBody extends React.PureComponent<Props, State> {
         let fileAttachmentHolder = null;
         if (
             ((post.file_ids && post.file_ids.length > 0) ||
-                (post.filenames && post.filenames.length > 0)) &&
+                (post.filenames && post.filenames.length > 0) ||
+                filePreviews.length > 0) &&
             this.props.post.state !== Posts.POST_DELETED
```

Comment:
this can be simplified and tidied up:

```tsx
if ((post.file_ids?.length || post.filenames?.length || filePreviews.length) && post.state !== Posts.POST_DELETED) {
    ...
}
```